tests/storage_pool/storage_pool_test.go

```diff
@@ -64,10 +102,10 @@ var _ = Describe("{StoragePoolResize}", func() {
 				}
 			}
 			Expect(newPool).ShouldNot(BeNil())
-			Expect(newPool.TotalSize).Should(Equal(newSize))
+			newTotalSize := newPool.TotalSize / units.GiB
+			Expect(newTotalSize).Should(BeNumerically(">=", oldSize/units.GiB))
```

Comment:
the same as above comment

Reply:
we can't do the exact size for the `disk-add`. If we run this test on a multi disk pool, the test will fail, because the test doesn't determine the disk size